### ISSUE TITLE
Feh is not setting the wallpapers automatically

### ISSUE THREAD

**Issue by tejas-luthra** [Keyitdev/dotfiles#3]
I am using Arch and I used the install script to get this up and running. I am only having one issue, feh is not setting up the wallpaper in both the lock screen and the home screen. When I am at the lockscreen, I see a black screen and when I am at the home screen, It is a grey screen. What should I do? I have reinstalled feh but it still doesn't work. 
Thanks,

**Comment by Keyitdev** [comment]
I am not 100% sure but I think it's a problem with path where the wallpapers are placed. Try setting the wallpaper manually
 `feh --bg-scale $HOME/Pictures/wallpapers/current_blue.png` (**make sure this path exists**) or any other path `feh --bg-scale ~/test.png`. If wallpaper sets up correctly, it's problem with path, if not, then we have to keep looking.

**Comment by Keyitdev** [comment]
Oh, now I see. I looked at the install script and it never copies the wallpapers to `$HOME/Pictures/wallpapers` it only creates that directory, my bad. Thanks for pointing that out. I will fix install script later. 
Run this to repair:
1. `mkdir -p $HOME/Pictures/wallpapers`
1. Go to localization where you cloned this repo.
1. `cp ./wallpapers/*  $HOME/Pictures/wallpapers/`

**Comment by tejas-luthra** [comment]
That works. Thanks. Also what files should I edit for config? ~/home/user/dotfiles/config or ~/home/user/dotfiles/config/dotfiles if it is none of these then what file do I edit to change the config? I know this might be an obvious question but I am new to this.

**Comment by Keyitdev** [comment]
Firstly, I don't know where exactly you cloned this repository. But it doesn't I will explain it anyways. Let say you cloned this repository in your home path (`~/dotfiles`). If you edit `~/dotfiles/config` you only change your local version of my repository. If you want to edit configs that  your system uses, you need to edit `~/.config/`. In this directory you will find the configs that are  used by installed applications.
In summary, if you want to edit something that will affect your system, you need to edit `~/.config/` for configs,  `/usr/local/bin` for scripts, `/usr/share/fonts/` for fonts, `~/Pictures/wallpapers` for wallpapers.
And by the way, `~`,  `/home/user` and `$HOME` are the same paths (unless you change it but that's for more advanced users). 
I hope I helped.

**Comment by tejas-luthra** [comment]
Thanks for helping out. I love this rice.
